python/bucketbase/ibucket.py

```diff
@@ -247,6 +249,16 @@ def get_object(self, name: PurePosixPath | str) -> bytes:
             self._unlock_object(name)
         return content
 
+    def get_size(self, name: PurePosixPath | str) -> int:
+        try:
+            self._lock_object(name)
```

Comment:
I'd say that we can try to return the size without locking the object, since this is an append only bucket, which would be sensibly faster. This one is used for caching, so the speed matters